spec/languages/lambda/convertAST-test.js

```diff
@@ -0,0 +1,61 @@
+import convertAST from 'codemirror-blocks/languages/lambda/convertAST';
+
+describe('the convertAST function for lambda to codemirror-blocks', function() {
+
+  it("should convert number to literal", function(){
+    this.AST = { type: "num", value: 123.5 };
+    this.cAST = convertAST(this.AST);
+    expect(this.cAST.rootNodes[0].dataType).toBe("number");
+    expect(this.cAST.rootNodes[0].value).toBe(123.5);    
+  });
+
+  it("should convert string to literal", function(){
+    this.AST = { type: "str", value: "Hello World!" };
+    this.cAST = convertAST(this.AST);
+    expect(this.cAST.rootNodes[0].dataType).toBe("string");
+    expect(this.cAST.rootNodes[0].value).toBe("Hello World!");
+  });
+
+  it("should convert bool to literal", function(){
+    this.AST = { type: "bool", value: true };
+    this.cAST = convertAST(this.AST);
+    expect(this.cAST.rootNodes[0].dataType).toBe("bool");
+  });
+
+  it("should convert identifiers to symbol", function() {
+    this.AST = { type: "var", value: "foo" };
+    this.cAST = convertAST(this.AST);
+    expect(this.cAST.rootNodes[0].dataType).toBe('symbol');
+    expect(this.cAST.rootNodes[0].value).toBe('foo');
+  });
+
+  fdescribe("when parsing function definitions,", function() {
```

Comment:
You should avoid checking in calls to `fdescribe()` or `fit()` since they disable all the other tests.